sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaRegistryClientLiveTests.cs

```diff
@@ -37,10 +39,26 @@ public async Task CanRegisterSchema()
             var format = SchemaFormat.Avro;
 
             SchemaProperties registerProperties = await client.RegisterSchemaAsync(groupName, schemaName, SchemaContent, format);
-            AssertSchemaProperties(registerProperties, schemaName);
+            AssertSchemaProperties(registerProperties, schemaName, format);
 
             SchemaProperties schemaProperties = await client.GetSchemaPropertiesAsync(groupName, schemaName, SchemaContent, format);
-            AssertSchemaProperties(schemaProperties, schemaName);
+            AssertSchemaProperties(schemaProperties, schemaName, format);
+            AssertPropertiesAreEqual(registerProperties, schemaProperties);
+        }
+
+        [RecordedTest]
+        public async Task CanRegisterSchemaJson()
```

Comment:
nit: consider adding TestCases to each of the existing tests and passing in the format as a parameter. I think this would make us re-record all tests because the test names will change.

Reply:
I just recorded the json test cases for now, the Avro test cases throw "Schema of type Avro cannot be created within group". I'm not sure if this is something that will be fixed once the json changes are all in on the service side, otherwise I may need to add additional test resources.